cbxtools/cli.py

```
@@ -1,981 +1,1371 @@
+
+    # Disable subcommand
+    disable_parser = subparsers.add_parser('disable', help='Disable statistics tracking', add_help=False)
+
+    # Add help argument last
+    parser.add_argument('-h', '--help', action='help', help='Show this help message and exit')
+
+
+
+
+
+def handle_convert_command(args, logger, stats_tracker=None):
```

Ruff (0.13.1)

Unused function argument: `args`

(ARG001)